content_creator.py

```diff
@@ -146,9 +148,23 @@ def copy_test_files(bundle_test):
             shutil.copyfile(path, os.path.join(bundle_test, os.path.basename(path)))
 
 
-def main(circle_artifacts):
-    print('Starting to create content artifact...')
+def update_content_version(key, new_value):
+    f = open('./Scripts/CommonServerPython/CommonServerPython.py', 'r')
+    lines = f.readlines()
+    f.close()
+    for i, line in enumerate(lines):
+        if line.split('=')[0].strip(' \n') == key:
+            lines[i] = f'{key} = "{new_value}"\n'
+    f = open('./Scripts/CommonServerPython/CommonServerPython.py', "w")
```

Comment:
No need to open the file twice, you can just open it once for both read and write

Reply:
there's no flag for doing so. if so - please let me know!